### PR TITLE
Add cgroupv2 variant to e2e installer

### PR DESCRIPTION
**Pull request by harche**
Signed-off-by: Harshal Patil <harpatil@redhat.com>

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -844,6 +844,35 @@ objects:
           popd
         fi
 
+        if has_variant "cgroupv2"; then
```
Comment (rphillips) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml:847]: cgroupsv2

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 2]
```diff
@@ -844,6 +844,35 @@ objects:
...
+          labels:
+            machineconfiguration.openshift.io/role: master
+          name: 99-openshift-cgroupsv2-master-kargs
+        spec:
+          kernelArguments:
+            - systemd.unified_cgroup_hierarchy=1
+            - cgroup_no_v1="all"
+            - psi=1
+        EOF
+          cat > ${manifests}/realtime-worker-machine-config.yml << EOF
```
Comment (rphillips) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml:861]: we should change the filenames to cgroupsv2 instead of realtime.

**Comment by kolyshkin** [comment]
I briefly checked the logs and from the first glance I don't see anything that sticks out to be cgroup v2 specific.

**Comment by umohnani8** [comment]
The rehearse job failures that I have seen so far are due to a secret not being found, I have seen that a few times in other places so most likely a flake. Don't believe they are cgroupsv2 related so this should be good to merge.
LGTM

**Comment by harche** [comment]
> The rehearse job failures that I have seen so far are due to a secret not being found, I have seen that a few times in other places so most likely a flake. Don't believe they are cgroupsv2 related so this should be good to merge.
> LGTM

Thanks @umohnani8

**Comment by rphillips** [comment]
@harche The job needs to be added to the config directory.

**Comment by harche** [comment]
Changing this PR to include only the variant for cgroupv2 so that https://github.com/openshift/ci-chat-bot/pull/182 can be unblocked.

**Comment by harche** [comment]
@stevekuznetsov @wking Need approval for this PR so that we can move forward with adding the support for cgroups v2 in the cluster bot using https://github.com/openshift/ci-chat-bot/pull/182

**Pull request merged**